bases/rsptx/interactives/runestone/common/js/bookfuncs.js

```diff
@@ -157,11 +157,13 @@ class PageProgressBar {
             subchapterprogress.max = 100;
             subchapterprogress.value = value;
         }
-        // Handle the case where there are no interactive objects on the page.
-        // but we still need to give the student points for reading the page.
-        if (this?.assignment_spec?.activities_required === 0) {
+        // If the user has completed all activities, send the reading score.
+        // This handles the case where there are no activities on the page or
+        //  where the user completed activities on the assignment page and now
+        //  is viewing the reading page.
+        if (this?.possible === this?.total) {
```

Comment:
What if the number of activities required is less than the total possible?  The 80% rule we typically create.  I think we might want to use the number required from the assignment_spec here???